Sources/Overload/OvRendering/include/OvRendering/Data/PipelineState.h

```diff
@@ -55,8 +56,9 @@ namespace OvRendering::Data
 				Settings::ERasterizationMode rasterizationMode : 2;
 
 				// B5
-				uint8_t lineWidthPow2 : 3;
-				Settings::EComparaisonAlgorithm depthFunc : 3;
+				uint8_t lineWidthPow2 : 2;
+				Settings::EComparaisonAlgorithm depthFunc : 2;
+				Settings::EMaterialDomain domain : 2;
```

Comment:
Since we only have 2 domains, we can use a single bit. Maybe the enum type should be updated too?
I'm woried about changing the `lineWidthPow2` from 3 bits to 2 bits, I don't think it gives us enough precision given the use cases of the `lineWidthPow2` value.